### DISCUSSION TITLE
Getting error on npm run dev

### DISCUSSION THREAD

**Discussion by Sapnajha19** [open-sauced-craftwork/browser-extension#38] [Q&A] [answered]
After setting up the project by following [CRXJS](https://crxjs.dev/vite-plugin/getting-started/vanilla-js/create-project) guide, when I do npm run dev. I got these errors
![manifest json - open_sauced_craftwork_browser_extension-Sapnajha19 - Visual Studio Code 22-03-2023 23_36_16](https://user-images.githubusercontent.com/99778524/226997934-edbe9b34-fe35-4a07-ba56-1fc531d9818c.png)

Also, I wanted to use React JS as the preferred framework, but the arrow keys were not selecting it. it was only selecting vanilla js.

**Comment by nickytonline** [comment, marked as answer]
Try deleting your `node_modules` folder for the project and run `npm install` again and then try running the project. Also, you may have more information in regards to the issue in the log file mentioned at the bottom of your screen shot.
